### DISCUSSION TITLE
How to ignore "Jupyter Notebooks" in "Languages activity" plugin?

### DISCUSSION THREAD

**Discussion by chbrandt** [lowlighter/metrics#1385] [Q&A]
I am trying to remove Jupyter Notebooks from the Language Activity plugin through the `plugin_languages_ignored` attribute but I couldn't figure out how exactly. I've tried "jupyter-notebook", "notebooks", "linguist-vendored" as values, but I still see "Jupyter Notebook" there:

![image](https://user-images.githubusercontent.com/482051/221231342-1c33b3f4-de53-4525-b2ff-6c250d65772e.png)

**Comment by lowlighter** [comment]
Hi 👋 !

Just using `plugin_languages_ignored: jupyter notebook` should work from what I've tested:
![image](https://user-images.githubusercontent.com/22963968/224597333-8990a927-c158-447d-86e9-ba6c295b2c66.png)

The split is done on commas, so spaces should be supported within the filtering
